skfda/misc/_math.py

```diff
@@ -677,3 +677,33 @@ def cosine_similarity_matrix(
     return _clip_cosine(
         inner_matrix / norm1[:, np.newaxis] / norm2[np.newaxis, :],
     )
+
+
+def function_to_fdatabasis(
```

Comment:
I would put it in `_utils` for now, as I am not sure that we want to expose it for final users at this point.